src/modules/ChannelList/stories/index.stories.js

```diff
@@ -2,7 +2,7 @@ import React, { useState, useCallback } from 'react';
 
 import Sendbird from '../../../lib/Sendbird';
 const appId = process.env.STORYBOOK_APP_ID;;
-const userId = 'sendbird';
+const userId = 'sendbird'; // Change this to 'sendbird1' for testing MultipleFilesMessage.
```

Comment:
I think this comment will be obsolete after allowing the selection of multiple files in the message input